README.md

```diff
@@ -17,8 +29,63 @@ For more information on Fixtures, check out this [pytest documentation](https://
 A **Mock** is an object that simulates the behavior of another object such as a class or function. They are used to simulate the behavior of real objects for testing purposes.
 For more information on Mocks, check out this [unittest documentation](https://docs.python.org/3/library/unittest.mock.html) on the mock module.
 
-**Example:**
+### Example
```

Comment:
Is it worth deleting this "Example"? It's a bit confusing having two"Example" sections in the README, and this one goes into a lot of technical detail. The usage example seems much more relevant and helpful to readers.

Reply:
Okay, I agree. I will remove this example section.